### DISCUSSION TITLE
Release plan / Release 1.26.4 / 1.27.1

### DISCUSSION THREAD

**Discussion by danielcb** [cri-o/cri-o#7114] [Q&A] [answered]
Hi,
I've tried to find any information regarding release plans. Are there any fixed times when new releases get tagged? Basically we're waiting for a release on the current release-1.26 and release-1.27 branches. Thanks!

**Comment by sohankunkerkar** [comment, marked as answer]
I agree that it would be beneficial to note the release timing in the documentation for better visibility and knowledge. CRI-O typically releases its latest version within one or two weeks after the release of Kubernetes. I think the minor version releases are a bit ad-hoc and mostly focused on bug fixes.

**Reply by haircommander** [reply]
FWIW: I have been planning a 1.27 and 1.26 one for a bit but there keep being changes I am trying to pull in. Is there a particular fix you're looking for?

**Reply by danielcb** [reply]
Hi, thanks for the answers. I'm waiting for the fix for issues regarding missing metrisc (#6657, #6805, etc.). The fix is already on both branches, but not yet released: https://github.com/cri-o/cri-o/pull/6856

**Reply by haircommander** [reply]
the two versions have been submitted today and should be packaged soon! thanks for your patience here
